src/QuickBooks/Provider.php

```diff
@@ -2,25 +2,87 @@
 
 namespace SocialiteProviders\QuickBooks;
 
-use SocialiteProviders\Manager\OAuth1\AbstractProvider;
-use SocialiteProviders\Manager\OAuth1\User;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+use Laravel\Socialite\Two\ProviderInterface;
+use SocialiteProviders\Manager\ConfigTrait;
 
-class Provider extends AbstractProvider
+class Provider extends AbstractProvider implements ProviderInterface
 {
+    use ConfigTrait;
+
     /**
      * Unique Provider Identifier.
      */
     const IDENTIFIER = 'QUICKBOOKS';
 
+    /**
+     * Sandbox endpoint for retrieving user info.
+     */
+    const USERSANDBOXENDPOINT = 'https://sandbox-accounts.platform.intuit.com/v1/openid_connect/userinfo';
```

Comment:
can we get some underscores in here to seperate words please :)

Reply:
no worries :)